test/unit/presenters/formats/local_transaction_presenter_test.rb

```diff
@@ -138,5 +147,38 @@ def result
       ]
       assert_equal expected, result[:routes]
     end
+
+    context "[:scotland_availability]" do
```

Comment:
These tests look good 👍  My only gripe is that we're only covering one administration. I'm just having a ponder what could be done to cover them all concisely.

We could change the context to be `context "devolved administration availability"` and then within that we could have a test that asserts that by default none of the nations are included (note `assert result[:details].has_key?(:wales_availability)` might be a slightly cleaner way to assert).

Then I wonder if for the next tests we could test one with Northern Ireland and keep the other as Scotland to have a little country variety.


I feel like this should be covering all of the administrations in a way

Reply:
ooh yeah! thanks! have updated.